libs/SmartSync/src/com/salesforce/androidsdk/smartsync/util/SmartSyncLogger.java

```diff
@@ -213,6 +213,16 @@ public static void v(String tag, String msg, Object obj) {
         v(tag, msg + ": " + objStr);
     }
 
+    /**
+     * Sets the log level to be used.
+     *
+     * @param level Log level.
+     */
+    public static void setLogLevel(SalesforceLogger.Level level) {
```

Comment:
Yup, will do. They could directly call SalesforceLogger with the right component name to achieve the same though, since it's a quasi-singleton.